pkg/vacuum/vacuum.go

```diff
@@ -46,15 +47,36 @@ func init() {
 }
 
 const (
-	sqlAcquireLock      = "SELECT _prom_catalog.lock_for_vacuum_engine()"
-	sqlListChunks       = "SELECT format('%I.%I', schema_name, table_name) FROM _ps_catalog.chunks_to_freeze WHERE coalesce(last_vacuum, '-infinity'::timestamptz) < now() - interval '15 minutes' LIMIT 1000"
-	sqlVacuumFmt        = "VACUUM (FREEZE, ANALYZE) %s"
-	sqlReleaseLock      = "SELECT _prom_catalog.unlock_for_vacuum_engine()"
-	delay               = 10 * time.Second
-	defaultDisable      = false
+	sqlAcquireLock       = "SELECT _prom_catalog.lock_for_vacuum_engine()"
+	sqlVacuumFmt         = "VACUUM (FREEZE, ANALYZE) %s"
+	sqlReleaseLock       = "SELECT _prom_catalog.unlock_for_vacuum_engine()"
+	sqlGetVacuumSettings = `
```

Comment:
why not `SELECT current_setting('vacuum_freeze_min_age'), current_setting('autovacuum_freeze_max_age')`?

Reply:
honestly, because i forgot that `current_setting` existed ;)